csi/server/plugin/opensds/controller.go

```diff
@@ -82,31 +81,21 @@ func GetDefaultProfile() (*model.ProfileSpec, error) {
 }
 
 // FindVolume implementation
-func FindVolume(req *model.VolumeSpec) (bool, bool, *model.VolumeSpec, error) {
-	isExist := false
+func FindVolume(req *model.VolumeSpec) (*model.VolumeSpec, error) {
 	volumes, err := client.ListVolumes()
-
 	if err != nil {
-		glog.Error("List volumes failed: ", err)
-
-		return false, false, nil, err
+		msg := fmt.Sprintf("List volumes failed: %v", err)
```

Comment:
error msg should start with lower case letter.

Reply:
Done